feature-configs/integration-kni1/sriov/sriov-networknodepolicy.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: sriov-network-node-policy
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: vfio-pci
+  isRdma: false
+  nicSelector:
+    pfNames:
+    - eno1
+    rootDevices:
```

Comment:
you can remove this part the PfName is good

Reply:
you mean removing the `rootDevices`?

Reply:
yes there is no need for it :)

Reply:
Done, thanks!